docs/admin/eventing/broker-configuration.md

```diff
@@ -25,27 +25,28 @@ data:
 
 ## Channel implementation options
 
-The following example shows a Broker object where the `spec.config` configuration is specified in a `config-br-default-channel` ConfigMap:
+The following example shows a Broker object where the `spec.config` configuration is specified in a `config-my-broker` ConfigMap:
 
 ```yaml
 apiVersion: eventing.knative.dev/v1
 kind: Broker
 metadata:
+  name: my-broker
+  namespace: my-namespace
   annotations:
     eventing.knative.dev/broker.class: MTChannelBasedBroker
-  name: default
 spec:
   # Configuration specific to this broker.
   config:
     apiVersion: v1
     kind: ConfigMap
-    name: config-br-default-channel
-    namespace: knative-eventing
+    name: config-my-broker
+    namespace: my-namespace
 ```
 
-A Broker object that does not have a `spec.config` specified uses the `config-br-default-channel` ConfigMap dy default because this is specified in the `config-br-defaults` ConfigMap.
+Unless configured otherwise in the aforementioned `config-br-defaults` ConfigMap, a Broker object that does not have a `spec.config` specified uses the `InMemoryChannel` implementation by default. This setting is inherited from the configuration specified in the `config-br-default-channel` ConfigMap.
```

Comment:
```suggestion
Unless a `default-br-config` is configured in the `config-br-defaults` ConfigMap, Brokers use the `InMemoryChannel` implementation by default. This setting is inherited from the configuration specified in the `config-br-default-channel` ConfigMap.
```

Reply:
I meant what I wrote, the ConfigMap that ships with Kn Eventing explicitly refers to the `InMemoryChannel` implementation. So unless configured otherwise (i.e. _unless the ConfigMap is modified by the user_), it will use this.
If ``default-br-config` is missing, Brokers don't work, so changing to the version you suggested is not an options.